sycl/include/CL/sycl/detail/backend_traits_opencl.hpp

```diff
@@ -188,6 +189,28 @@ template <> struct InteropFeatureSupportMap<backend::opencl> {
   static constexpr bool MakeKernel = true;
   static constexpr bool MakeKernelBundle = true;
 };
+
+namespace pi {
+// Cast for std::vector<cl_event>, according to the spec, make_event
+// should create one(?) event from a vector of cl_event
+template <class To> inline To cast(std::vector<cl_event> value) {
+  RT::assertion(value.size() == 1,
+                "Temporary workaround requires that the "
+                "size of the input vector for make_event be equal to one.");
+  return (To)(value[0]);
```

Comment:
nit: Should we use `cast` here to assert for the size requirements?

Reply:
Good call! This has been changed.